src/ExportMap.js

```diff
@@ -296,442 +732,6 @@ function captureTomDoc(comments) {
 
 const supportedImportTypes = new Set(['ImportDefaultSpecifier', 'ImportNamespaceSpecifier']);
```

Comment:
this (and i think the others) is used inside parse, so let's move all such things higher than the class definition?

or, is there's a conceptual cycle here so there's no source position that can fully satisfy "define before use"?

Reply:
I wasn't aware that we have a convention of "define before use".
CI passed, so it's not enforced in lint. If we have that convention, then I believe we ought to enforce it in lint.

Reply:
it's enforced for functions - but i guess the eslint core rule doesn't apply to const/lets.